### PR TITLE
Accept multiple prefixes

### LINKED ISSUE #12: Accept multiple prefixes
## Feature request

The CLI should be able to accept an arbitrary number of prefixes.

### Current behavior:
```sh
npx print-env FOO BAR
# Only prints env variables starting with FOO
```

### Desired behavior:
```sh
npx print-env FOO BAR
# Prints env variables starting with FOO or BAR
```

### PR DESCRIPTION
**Pull request by examosa**
Closes #12

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,8 +9,6 @@ node_js:
 script:
   - npm test
   - npm run travis-demo
-  # print names of all sensitive variables present
-  - ./bin/has-env.js NPM
```
Comment (bahmutov) [inline .travis.yml:13]: try variable prefix that's for sure present

- ./bin/has-env.js TRAVIS

**Comment by bahmutov** [comment]
I love it overall

**Comment by bahmutov** [comment]
@Teomik129 I know this is probably a small thing, but would you like to be a collaborator on this repo? I am very short on time these days and it might be faster if you could review and merge your own PRs

File: src/print-env.spec.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+  it('supports multiple prefixes', () => {
+    snapshot(printEnv(['FOO', 'BAR']))
+  })
+
+  describe('bin print-env command', () => {
+    const bin = join(__dirname, '..', 'bin', 'print-env.js')
+    it('prints sorted variables', async () => {
+      const { stdout } = await execa('node', [bin, 'FOOX'], {
+        env: process.env
```
Comment (bahmutov) [inline src/print-env.spec.js]: we can use https://github.com/sindresorhus/execa#extendenv option to only pass `env` object and not the entire `env + process.env`

File: src/print-env.spec.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+const clone = x => JSON.parse(JSON.stringify(x))
+const snapshot = require('snap-shot-it')
+const execa = require('execa')
+const { join } = require('path')
+
+describe('@bahmutov/print-env', () => {
+  let env
+
+  beforeEach(() => {
+    env = clone(process.env)
```
Comment (bahmutov) [inline src/print-env.spec.js]: we can either use `extendEnv` (see below) or https://github.com/bahmutov/mocked-env to mock the environment object

**Comment by examosa** [comment]
> @Teomik129 I know this is probably a small thing, but would you like to be a collaborator on this repo? I am very short on time these days and it might be faster if you could review and merge your own PRs

I would be more than happy to be a collaborator for this project!

**Review by bahmutov — DISMISSED** [review]
need to explain new feature in README

File: .travis.yml (hunk 1 above)
Comment (examosa) [reply .travis.yml:13]: Fixed

File: src/print-env.spec.js (hunk 3 above)
Comment (examosa) [reply src/print-env.spec.js]: Node processes can't permanently alter the environment, so I believe we can safely overwrite `process.env` without having to restore it later.

File: src/print-env.spec.js (hunk 2 above)
Comment (examosa) [reply src/print-env.spec.js]: Better yet, I just added to `process.env` and didn't pass any options to `execa`; see next commit.

**Review by examosa — COMMENTED** [review]
All good

**Pull request merged**